app/locales/en/translations.js

```diff
@@ -0,0 +1,233 @@
+export default {
```

Comment:
I'd like to see us build up a common dictionary, then modularize the translations eventually. That would reduce the workload on the people actually doing the translations.